lms/djangoapps/certificates/models.py

```diff
@@ -1264,3 +1264,43 @@ class Meta:
 
     def __str__(self):
         return str(self.arguments)
+
+@python_2_unicode_compatible
```

Comment:
(What is this? Do we need it? This is on the `CertificateInvalidation` model, which I’m loosely referencing.)

Reply:
We don't need this anymore. It was a pre-python3 thing.

Reply:
Amazing thank you ✅